spec/PeriodicBackgroundSync-index.bs

```diff
@@ -79,3 +80,18 @@ The options are a dictionary containing [=minInterval=], a long long. <dfn>minIn
 Enclosing options in a dictionary allows this spec to be extended with more options in the future without adversely affecting existing usage.
 
 The <dfn>registration state</dfn> is one of <dfn>pending</dfn>, <dfn>waiting</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+
+Privacy Considerations {#privacy}
+==================================
+
+## Permission ## {#permission}
+Periodic Background Sync is only available if the {{PermissionState}} for a {{PermissionDescriptor}} with {{name}} <code>"periodic-background-sync"</code> is {{granted}}.
+
+In addition, user agents SHOULD offer a way for the user to disable periodic background sync.
+
+## Location Tracking ## {#location-tracking}
+Fetch requests within the <code>periodicsync</code> event while [=in the background=] may reveal the client's IP address to the server after the user has left the page. The user agent SHOULD limit tracking by capping the number of retries and duration of <code>periodicsync</code> events, to reduce the amount of time the user's location can be tracked by the website.
+Further, the user agent SHOULD limit persistent location tracking by capping the frequency of <code>periodicsync</code> events, both for an origin, and across origins.
```

Comment:
nit: either have line breaks (this section), or don't (the next section), but avoid having both